content/en/docs/reference/glossary/manifest.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Manifest
+id: manifest
+date: 2019-06-28
+short_description: >
+  A serialized definition of one or more Kubernetes API objects.
```

Comment:
Maybe s/definition/specification/. I think the key here is that it's a description of desired state without any status. Definition is conflated with structure due to CRDs.

Reply:
I have replaced definition with specification.